pkg/reporting/nodes.go

```diff
@@ -18,20 +18,51 @@ package reporting
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 )
 
-// CookbookVersion name and version
+// CookbookVersion name and version (sortable by name)
 type CookbookVersion struct {
 	Name    string
 	Version string
 }
 
+// Sort interface the sorts cookbooks by name and version.
+type CookbookByNameVersion []CookbookVersion
+
+func (cnv CookbookByNameVersion) Len() int {
+	return len(cnv)
+}
+func (cnv CookbookByNameVersion) Swap(i, j int) {
+	cnv[i], cnv[j] = cnv[j], cnv[i]
+}
```

Comment:
I wish go made these to function implicit. sucks to redefine the sam thing every time.